File: 2-semestre/uri-online-judge/javascript/1177.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

const t = parseInt(lines.shift());
let arr = [], c = 0;

for (let i = 0; i < 1000; i++) {
    arr[i] = c;
    c++;

    if (arr[i] === t-1) c = 0;

    console.log(`N[${i}] = ${arr[i]}`);
}

File: 2-semestre/uri-online-judge/c/1177.c
#include <stdio.h>
 
int main() {
    int t, i, c = 0, arr[1000];

    scanf("%d", &t);

    for (i = 0; i < 1000; i++) {
        arr[i] = c;
        c++;

        if (arr[i] == t-1) c = 0;

        printf("N[%d] = %d\n", i, arr[i]);
    }

    return 0;
}
